src/Microsoft.DotNet.Interactive.CSharpProject/Packaging/PackageInitializer.cs

```diff
@@ -67,4 +73,32 @@ private static string GetLanguageFromProjectName(string projectName)
         // default to C#
         return "C#";
     }
+
+    private async Task CreateGlobalJsonAsync(Dotnet dotnet, string sdkVersion = null)
+    {
+        if (sdkVersion is null)
+        {
+            if (_defaultSdkVersion is null)
+            {
+                var resultListAvailableSdks = await dotnet.ListSdks();
+                _defaultSdkVersion = GetLatestVersionBeforeEight(resultListAvailableSdks.Output);
+            }
+
+            sdkVersion = _defaultSdkVersion;
+        }
+
+        await dotnet.NewGlobalJson(sdkVersion);
+    }
+
+    private string GetLatestVersionBeforeEight(IEnumerable<string> versions)
+    {
+        var maxVersion = new Version(8, 0);  // specify the upper limit version
```

Comment:
Is this going to do something different from what we can do with `dotnet new globaljson`?

Reply:
Removed.

Added the suggested change to create a global.json with latest sdk